src/components/Header.js

```diff
@@ -1,11 +1,33 @@
-import React from "react";
+import React, { useState } from "react";
+import SignIn from "../pages/SignIn";
 
 const Header = () => {
-    return (
-        <div>
-            <h1>Header</h1>
-        </div>
-    );
-}
+  return (
+    <React.Fragment>
```

Comment:
`<></>` are the same as `<React.Fragment></React.Fragment>`. Mostly seen as <></> when in use.